tests/e2e/e2e_suite_test.go

```diff
@@ -290,27 +290,6 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 	SetDefaultEventuallyTimeout(240 * time.Second)
```

Comment:
I think these eventually defaults need removing, as they were added just for this pre-push step

Reply:
Thanks, I did not realised that those are no longer needed.